server/index.html

```diff
@@ -3,38 +3,146 @@
+          <div class="prompts-wrapper">
+            <section class="prompts-players">
+              <h2 class="prompts-player-header">Players</h2>
+              <div class="prompts-wrapper-inner">
+                <label class="prompts-radio-label" for="player-num-1">1</label>
+                <input class="prompts-radio" type="radio" id="1-player" name="player-num" value="1-player" checked>
+
+                <label class="prompts-radio-label" for="player-num-2">2</label>
+                <input class="prompts-radio" type="radio" id="2-player" name="player-num" value="2-player">
+              </div>
+            </section>
+            <section class="prompts-team">
+              <h2 class="prompts-team-header">Team</h2>
+              <div class="prompts-wrapper-inner">
+                <label class="prompts-radio-label" for="team-o">O</label>
+                <input class="prompts-radio" type="radio" id="team-o" name="player-team" value="team-o" checked>
+
+                <label class="prompts-radio-label" for="team-x">X</label>
+                <input class="prompts-radio" type="radio" id="team-x" name="player-team" value="team-x">
+              </div>
+            </section>
+          </div>
+          <div class="prompts-submit">
+            <button
+            type="button" class="prompts-form-submit" onClick="startGame()">Start Game!</button>
+          </div>
+        </form>
+      </div>
+
+      <div class="game-header" style="display: flex; justify-content: center;">
+        <h2 class="game-header-player" id="current-player" style="margin-right: 50px;">Current Player</h2>
+        <h2 class="game-header-team" id="current-team">Team</h2>
+      </div>
+      <div class="game-grid-wrapper">
+        <div class="game-grid disabled" id="game-grid">
+          <div class="game-grid-item" id="grid-1">1</div>
+          <div class="game-grid-item" id="grid-2">2</div>
+          <div class="game-grid-item" id="grid-3">3</div>
+          <div class="game-grid-item" id="grid-4">4</div>
+          <div class="game-grid-item" id="grid-5">5</div>
+          <div class="game-grid-item" id="grid-6">6</div>
+          <div class="game-grid-item" id="grid-7">7</div>
+          <div class="game-grid-item" id="grid-8">8</div>
+          <div class="game-grid-item" id="grid-9">9</div>
+        </div>
+        <div class="game-over hidden" id="game-over">
+          <h2 class="game-over-header">GAME OVER</h2>
+          <h3 class="game-over-winner" id="game-winner">Someone won!</h3>
+          <button type="button" onClick="playAgain()">Play Again?</button>
+        </div>
+      </div>
     </div>
-  </div>
-    <div class="game-log">
-      <p class="game-log-text" id="logs" style="display: block">Logs go here</p>
+
+    <div class="log-wrapper">
+      <div class="log">
+        <h2 class="log-header">Game Logs</h2>
+        <div class="log-text-container" id="log-container">
+          <p class="log-text" id="logs">Logs appear here</p>
```

Comment:
I noticed that if the websocket is disconnected (the backend websocket server isn't running) you only get a log in the devtools pane. Would be nice to also get a log here.

Reply:
As a first stab, it would probably enough to add a try/catch around `const ws = new WebSocket("ws://localhost:8080");` and add a "Could not connect to websocket. Did you forget to run `yarn node server/index.js`?" message in the log box if it fails (and then re-throw the error).